Overview of the potential development options about the OPC UA Bindings (#1105)
w3c/wot: liaisons/opcf/collaboration-options.md

There are several options how the development of the OPC UA Binding for the Web of Things can be managed:

# As W3C Note

> W3C members setup an OPC UA Binding document as W3C Note. OPC UA experts can be involved for getting feedback.

Reviewer: ```suggestion
W3C members setup an OPC UA Binding document as W3C Note. OPC UA experts can be involved for getting feedback.
OPC UA *may* later adopt it or modify it as a companion specification.
```

Pro

> * alligned with existing approaches such as with Modbus Binding

Reviewer: ```suggestion
* aligned with existing approaches such as Modbus Binding
```

Cons

> * no testing / conformance testing provided

Reviewer: ```suggestion
* no testing / conformance testing provided
* potential for adopted OPC UA companions specification to diverge from W3C version
```

> # Develop W3C Note, transformed it later to W3C REC

Reviewer: ```suggestion
# Develop W3C Note, transform it later into a W3C REC
```

> A previos developed W3C Note about OPC UA Binding can be used or transformed to a REC. OPC UA experts can be involved for getting feedback. 

Reviewer: ```suggestion
A previously developed W3C Note about OPC UA Binding can be used or transformed to a REC. OPC UA experts can be involved for getting feedback. 
OPC UA *may* later adopt it or modify it as a companion specification.
```

Pro

> *  includes testing

Reviewer: ```suggestion
* includes testing
* the decision to convert from a Note to a Rec will require AC review, which (if successful) will give the document more of a mandate from W3C as a whole
* assertions and RFC2119 keywords will be more appropriate
```

Cons
* still no conformance testing provided

> * requires more time 

Reviewer: ```suggestion
* requires more time and effort: need to publish Note, then change charter, go through Rec process, etc.
* potential for adopted OPC UA companions specification to diverge from W3C version
```

# As W3C REC only

> W3C members setup an OPC UA Binding document as W3C Recommendation. OPC UA experts can be involved for getting feedback in that work. 

Reviewer: ```suggestion
W3C members setup an OPC UA Binding document as W3C Recommendation. OPC UA experts can be involved for getting feedback in that work. 
OPC UA *may* later adopt it or modify it as a companion specification.
```

Pro

> *  includes testing

Reviewer: ```suggestion
* includes testing
* as above, "mandate" and "assertive language" benefits
```

Cons

> * still no conformance testing provided

Reviewer: ```suggestion
* still no conformance testing provided
* extra time still required to change the charter (but potentially parallelizable with other work - can start work under assumption will be published as a Rec)
* potential for adopted OPC UA companions specification to diverge from W3C version
```

# As OPC UA Companion Standard only
OPCF Members setup an OPC UA Binding document as OPC UA Companion. W3C WoT experts can be involved for getting feedback.

Pro

> *  includes testing / conformance

Reviewer: ```suggestion
*  conformance testing organized by OPC UA according to their needs.
```

> *  follows the new de-coupling binding approache of the new WoT charter 

Reviewer: ```suggestion
*  follows the new de-coupling binding approach of the new WoT charter 
* avoids divergence: single document managed by a single organization
```

> *  follows the new de-coupling binding approache of the new WoT charter 

Reviewer: Cons
* Binding is not under control of the WoT WG and will require coordination to ensure it follows a consistent template, document structure, is consistent with the TD specification, etc.

Author: ok, I will add this
